### PR TITLE
 Transaction support to the CSM load test.

### PR DESCRIPTION
**Pull request by alawibaba**

### CODE REVIEW

**Comment by alawibaba** [comment]
@cpennington PTAL

File: csm/locustfile.py [hunk 1]
```diff
@@ -184,10 +185,12 @@ def get_many(self):
             )
 
     @task(1)
+    @transaction.commit_manually
     def set_many(self):
         usage_key = self._gen_usage_key()
         self.client.set_many(self.client.username, {usage_key: self._gen_block_data()})
         self.usages_with_data.add(usage_key)
+        transaction.commit()
```
Comment (cpennington) [inline csm/locustfile.py]: In your tests in devstack, did the transaction that had the `set` in it also have `get`s?

Comment (cpennington) [inline csm/locustfile.py]: I would have expected at least one call to `get` to retrieve the data that's being `set`.

**Comment by cpennington** [comment]
:+1:

**Pull request closed without merging**